app/views/declaration/transport_information.scala.html

```diff
@@ -37,6 +37,10 @@
 
         @components.page_title(Some("supplementary.transportInfo.title"), None)
 
+
+
+@helper.select(field = form("inlandModeOfTransportCode"), options = Seq("foo"->"Foo", "bar"->"Bar"))
```

Comment:
this Foo and Bar should be here or you use it only for testing?

Reply:
will remove, I was using it for testing.